tests/std/tests/P0608R3_sane_variant_converting_constructor/test.cpp

```diff
@@ -93,13 +99,16 @@ void test_variant_constructor_P0608R3() {
     using T2 = variant<float, long>;
     T2 e;
     assert(e.index() == 0);
+#ifndef __EDG__ // TRANSITION, ...
     e = 0; // long
+#endif // !__EDG__
     assert(e.index() == 1);

```

Comment:
```suggestion
    assert(e.index() == 1);
#endif // !__EDG__
```
If `e = 0;` is removed, the assert should be removed to avoid assertion failure on `EDG`